File: laba 1 sem/laba 4/laba 4 py/4.4/3.py
from Lenght import lenght
text = input("Введите предложение:  ")
i = 0
while i < lenght(text):
    if text[i] == "c" or text[i] == "C":
        print("c была раньше")
        break
    if text[i] == "t" or text[i] == "T":
        print("t была раньше")
        break
    i += 1

File: laba 1 sem/laba 4/laba 4 pas/4.3/3.pas
function lenght(str: string): integer;
  var 
    i: integer;
    s: string;
  begin    
    i := 1;
    while s <> str do
      begin
      s += str[i];
      i += 1
    end;
  Result := i - 1;
end;

var
  text: string;
  x: Integer;

begin
  Write('Введите предложение: ');
  ReadLn(text);

  for x := 2 to length(text) do
    if (x) mod 3 = 0 then
      WriteLn(text[x]);
end.